linux/example/Makefile

```diff
@@ -75,7 +76,7 @@ $(FLUTTER_EXAMPLE_DIR)/build:
 		--local-engine=$(FLUTTER_ENGINE_BUILD);
 
 $(BIN_OUT): $(SOURCES) $(LIBRARIES)
-	$(CXX) $(SOURCES) $(CXXFLAGS) -o $@
+	$(CXX) $(SOURCES) $(CXXFLAGS) $(CPPFLAGS) $(LDFLAGS) -o $@
```

Comment:
Let's keep C*FLAGS before the sources list.

Reply:
Done